### ISSUE TITLE
show timer instead of % in center of circle

### ISSUE THREAD

**Issue by hupptech** [lzyzsd/CircleProgress#46]
I'm trying to show countdown type timer like :  01:30 or 03:50 it's mm:ss but now sure what property to use for it.
I only have option to set prefix and suffix but not the actual text. I have total time for countdown timer in seconds and I can also calculate it to minutes and second, but how do i display it in center ?

If i don't set prefix and suffix it takes 0 by default which I don't want to show.

**Comment by padobariya** [comment]
@hupptech did you found any solution ?

**Comment by hupptech** [comment]
No. I ended up doing the custom circuclar progress bar myself. Frankly speaking that was quiet easy for me to do compared to finding a fix in this library. @padobariya

**Comment by prashanthd** [comment]
@hupptech  can you please guide me to achieve that functionality. If you share code that would be more helpful.
